pkg/performanceprofile/controller/performanceprofile_controller.go

```diff
@@ -60,9 +60,9 @@ import (
 )
 
 const (
-	openshiftFinalizer               = "foreground-deletion"
-	hypershiftFinalizer              = "hypershift.openshift.io/foreground-deletion"
-	controllerGeneratedMachineConfig = "hypershift.openshift.io/performanceprofile-config"
+	openshiftFinalizer                    = "foreground-deletion"
+	hypershiftFinalizer                   = "hypershift.openshift.io/foreground-deletion"
+	ControllerGeneratedPerformanceProfile = "hypershift.openshift.io/performanceprofile-config"
```

Comment:
Let's move it to hypershift package because we'll have to import it from here and it will cause cyclic dependencies

Reply:
On second thought let's leave it here and move it in future PR when we'll actually need it somewhere else